arch-install.bash

```diff
@@ -269,19 +291,35 @@ useradd -m -G wheel -p "$hashed_password" -- "$username"
 
 printf '%s\n' '%wheel ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
 
+# home dir
 home=$(getent passwd -- "$username" | awk -v RS='' -F ':' '{ print $6 }')
 pushd "$home"
 git clone https://aur.archlinux.org/yay.git
 pushd yay
 chown -R -- "$username:$username" .
 runuser -u "$username" -- makepkg -si --noconfirm
 popd
+
+if [[ -v git_repo ]]; then
+  git clone "$git_repo"
+  pushd "$(basename "$git_repo" .git)"
+  echo $PWD
+  IFS=';'
+  for encoded_arg in $encoded_command; do
+    command+=("$(base64 --decode <<<"$encoded_arg")")
+  done
+  "${command[@]}"
+  unset IFS
```

Comment:
After looking at it in detail, I think using "$IFS" for this is ugly.

I suggested above to use the `escape` function to encode the command, so we might as well use the `unescape` function to decode it back into an array:

```suggestion
  command=()
  while read -r line; do
    command+="$(unescape "$line")"
  done <<< "$encoded_command"
```

We would need to export the function (via `export -f unescape`) to the bash process running in the chroot for this to work.